docker/lite/build.sh

```diff
@@ -3,15 +3,51 @@
 # This is the script to build the vitess/lite Docker image by extracting
 # the pre-built binaries from a vitess/base image.
 
+# Parse command line arguments.
+prompt_notice=true
+if [[ "$1" == "--prompt"* ]]; then
+  if [[ "$1" == "--prompt=false" ]]; then
+    prompt_notice=false
+  fi
+  shift
+fi
+
 flavor=$1
 
+tag=latest
 if [[ -n "$flavor" ]]; then
   base_image=vitess/base:$flavor
+  tag=$flavor
 else
   echo "Flavor not specified as first argument. Building default image."
   base_image=vitess/base
 fi
 
+# Abort if base image does not exist.
+if ! docker inspect $base_image &>/dev/null; then
+  echo "ERROR: Dependent image $base_image does not exist. Run 'make docker_base' to build it locally or 'docker pull $base_image' to fetch it from Docker Hub."
+  exit 1
+fi
+
+# Educate the user that they have to build or pull vitess/base themselves.
+if [[ "$prompt_notice" = true ]]; then
+  cat <<END
+
+This script is going to repack and copy the existing *local* base image '$base_image' into a smaller image 'vitess/lite'.
+
+The 'docker images' output below shows you how old your local base image is:
+END
+  docker images vitess/base | grep -E "(CREATED|$tag)"
```

Comment:
I believe you can put this into "$()", i.e. like this
<code>
$(docker images vitess/base | grep -E "(CREATED|$tag)")
</code>
Then you won't need to end one "cat" and start another one.